slides/src/content/services.md

```diff
@@ -114,4 +114,137 @@ import { HttpModule } from '@angular/http';
 export class AppModule {}
 ```
 
-[View Example](https://plnkr.co/edit/E9xJ9nUGxOQ66AbOmGXL?p=preview)
\ No newline at end of file
+[View Example](https://plnkr.co/edit/E9xJ9nUGxOQ66AbOmGXL?p=preview)
+
+---
+
+## Handling `http` rejections
+
+To catch rejections we can use the `subscribe` operator's `error` and `complete` callbacks
+
+```ts
+  this.http.post(`${ BASE_URL }/auth/login`, payload)
+    .map(response => response.json())
+    .subscribe(
+      (authData) => this.storeToken(authData.id_token), // Next function
+      (err) => console.error(err),                      // Error function
+      () => console.log('Authentication Complete')      // Complete function
+    );
+```
+
+---
+
+## Using the `.catch` operator
+
+- The `.catch` operator allows us to catch errors on a stream, do something, then pass the exception onwards
+- This also means we can inspect the error and decide how to handle it
+
+```ts
+  return this.http.get('https://api.spotify.com/v1/dsds?q=' + term + '&type=artist')
+    .map((response) => response.json())
+    .catch(err => {
+      if (err.status >==  500) {
+        return cachedVersion();
+      } else {
+        return Observable.throw(
+          new Error(`${ err.status } ${ err.statusText }`)
+        );
+      }
+    });
+```
+[View Example](http://plnkr.co/edit/3lCaeI?p=preview)
+
+---
+
+## Cancel a Request
+
+Cancelling an HTTP request is a common requirement. If you have a queue of requests and a new
+request supersedes the pending requests, you can call `unsubscribe` on the pending request's subscription.
```

Comment:
Cancelling an HTTP request is not only common but one of the real benefits of using observables instead of promises. I get that question a lot during training: "Why do we need to use observables if we can do the same thing with promises?"

We should try to emphasize that